google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/JsonToProtoMessage.java

```diff
@@ -235,7 +239,14 @@ private static void fillRepeatedField(
         case BYTES:
           if (val instanceof String) {
             protoMsg.addRepeatedField(fieldDescriptor, ((String) val).getBytes());
+          } else if (val instanceof JSONArray) {
```

Comment:
A byte array becomes a type of JSON Array is a bit strange. In your case, you happen to have a byte array in the JSON array, but what if you don't have byte array in the json array?

Reply:
That's a good point, I will add a check for the array being full of bytes and if it isn't maybe we should have a unique error message thrown. I'll also add a test that expects this error in the JsonToProtoMessageTest file.

Reply:
I made the check so that if anything in the array doesn't behave nicely, or convert to a byte without giving an error, the default error message is thrown stating that the instance does not have the correct type. This is now covered in the JsonToProtoMessageTest as well.